data/quarg/quarg outfits.txt

```diff
@@ -139,5 +143,5 @@ outfit "Quantum Shield Generator"
 	"mass" 120
 	"outfit space" -120
 	"shield generation" 10
-	"shield energy" 10
+	"shield energy" 5
```

Comment:
Considering how ridiculous powerful Antimatter Cores are, I feel a reduction in energy usage here isn't actually going to help them. I'd rather reduce the mass or increase the shield generation to bring these above Dark Storms and Overcharged Shield Modules, or give them some other attribute.
![image](https://user-images.githubusercontent.com/15916854/200365199-f21aa4c1-c8e1-4579-8345-3b12ca2db546.png)

Reply:
Maybe they eat burning?